src/integration/globalization.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const eachRegTopic = require('../helpers/registryIterator'),
+  replacer = require('../helpers/replacer'),
+  copier = require('../helpers/copier'),
+  misc = require('../helpers/misc'),
+  pathCreator = require('../helpers/pathCreator'),
+  async = require('async'),
+  log = require('../helpers/logger');
+  
+
+function globalization(registry, config, mapMarketsToRegions, next) {
+  eachRegTopic.async(registry, config, next, (topicDetails, cb) => {
+    const regions = mapMarketsToRegions(topicDetails.markets);
```

Comment:
its mapMarketToRegion is provided from outside of chewie, we should provide somewhere 'schema' of the return value - how regions should looks like. Maybe for now - at least as a comment?